lms/djangoapps/oef/views.py

```diff
@@ -0,0 +1,81 @@
+import json
+
+from django.contrib.auth.decorators import login_required
+from django.core.urlresolvers import reverse
+from django.http import JsonResponse
+from django.shortcuts import render, redirect
+from rest_framework import status
+
+from lms.djangoapps.oef.helpers import *
+
+
+@login_required
+def oef_dashboard(request):
+    user_surveys = UserOefSurvey.objects.filter(user_id=request.user.id)
+    surveys = []
+    for survey in user_surveys:
+        surveys.append({
```

Comment:
Please use mako templates instead of django template engine. After that you can send objects to template and apply date format in templates instead of iterating records here